inference-engine/src/vpu/common/src/ngraph/query_network.cpp

```diff
@@ -66,11 +66,11 @@ InferenceEngine::QueryNetworkResult getQueryNetwork(const InferenceEngine::CNNNe
         }
     }
 
-    for (const auto& layerName : supported) {
+    for (const auto& layerName : unsupported) {
         if (supported.empty()) {
             break;
         }
-        if (InferenceEngine::details::contains(unsupported, layerName)) {
+        if (InferenceEngine::details::contains(supported, layerName)) {
```

Comment:
Do we really need this check? 
> size_type erase( const key_type& key );
> Removes the element (if one exists) with the key equivalent to key.

Reply:
it looks like this check isn't needed, I'll remove it